.vscode/settings.json

```diff
@@ -3,14 +3,13 @@
     "source.fixAll.tslint": true
   },
   "editor.formatOnSave": true,
-
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "[javascript]": {
     "editor.defaultFormatter": "esbenp.prettier-vscode"
   },
-
   "python.linting.enabled": true,
   "python.linting.lintOnSave": true,
+  "python.linting.mypyEnabled": true,
```

Comment:
allows (but does not enforce) static typing in python functions

Reply:
Very lyft of you